sql/dql/basic-queries/where-2.md

```diff
@@ -59,22 +59,22 @@ WHERE name = 'Meowth'
   OR name = 'Golduck';
 ```
 
-This shows the rows where the `name` is an exact match of either 'Meowth' or 'Golduck'. The resulting set looks like this:
+This shows the rows where the `name` is an exact match of either `'Meowth'` or `'Golduck'`. The resulting set looks like this:
```

Comment:
Just one more mention, this is the correct usage of both ` ` and `" "` (or `' '`) in the same sentence 👍